app/src/main/java/com/mobile/tandil/javabaseproject/activity/MainActivity.java

```diff
@@ -0,0 +1,24 @@
+package com.mobile.tandil.javabaseproject.activity;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+import com.mobile.tandil.javabaseproject.R;
+import com.mobile.tandil.javabaseproject.mvp.model.ParkingModel;
+import com.mobile.tandil.javabaseproject.mvp.presenter.ParkingPresenter;
+import com.mobile.tandil.javabaseproject.mvp.view.ParkingView;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        new ParkingPresenter(new ParkingModel(), new ParkingView(this));
+
```

Comment:
remove this extra space please, never put an empty line before close curly brackets